join-git-repos.py

```diff
@@ -408,6 +408,7 @@ def mergerpos(main_commands, secondary_commands, main_spec, secondary_spec):
         processed_all_commands = True
         first_commit_of_branch = (source['idx'] == 0)
         mark_before_break = b''
+        expectingTagFromMark = False
```

Comment:
Nitpick... Style: `expecting_tag_from_mark`

Reply:
Whoops! I corrected it to follow your naming convention.